app/assets/javascripts/discourse/tests/acceptance/sidebar-categories-section-test.js

```diff
@@ -149,6 +150,31 @@ acceptance("Sidebar - Categories Section", function (needs) {
     );
   });
 
+  test("category section links are sorted by category name alphabetically", async function (assert) {
+    const { category1, category2 } = setupUserSidebarCategories();
+
+    category2.set("name", "abc");
```

Comment:
Minor nitpick but I would add a couple more categories with names including uppercase characters just to make sure they sort correctly too, but otherwise this looks great :+1:

Reply:
Good idea :) Done.